packages/netsuite-adapter/src/client/suiteapp_client/suiteapp_client.ts

```diff
@@ -247,7 +254,13 @@ export default class SuiteAppClient {
     operation: RestletOperation,
     args: Record<string, unknown> = {}
   ): Promise<unknown> {
-    const response = await this.safeAxiosPost(this.restletUrl.href, { operation, args }, this.generateHeaders(this.restletUrl, 'POST'))
+    const response = await this.safeAxiosPost(
+      this.restletUrl.href,
+      this.useSignatureInRestletRequests
+        ? { operation, args, signature: this.credentials.accountIdSignature }
+        : { operation, args },
```

Comment:
styling nit, you can ignore if you prefer yours..
```suggestion
      {
        operation,
        args,
        ...{ this.useSignatureInRestletRequests ? signature: this.credentials.accountIdSignature : {} },
      },
```